### PR TITLE
fix: Change env variables according to cht pipeline updates

### PR DESCRIPTION
**Pull request by andrablaj**
Update env variables according to [cht-pipeline/pull/73](https://github.com/medic/cht-pipeline/pull/73). 

cht-pipeline/pull/73 needs to be merged before merging this PR. 

The CI will fail until the PR above is merged.

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -7,19 +7,19 @@ jobs:
     name: E2E Tests
     runs-on: ubuntu-latest
     env:
-      POSTGRES_USER: "root"
-      POSTGRES_PASSWORD: "supercoolpassword"
+      POSTGRES_USER: "postgres"
```
Comment (lorerod) [inline .github/workflows/test.yml:10]: Using this value and running the test locally, I got this error on Postgres container:
```
2024-02-23 08:38:26 2024-02-23 11:38:26.877 UTC [61] FATAL:  password authentication failed for user "root"
2024-02-23 08:38:26 2024-02-23 11:38:26.877 UTC [61] DETAIL:  Role "root" does not exist.
2024-02-23 08:38:26     Connection matched pg_hba.conf line 99: "host all all all md5"
```

File: env.template [hunk 2]
```diff
@@ -2,23 +2,23 @@
 COMPOSE_PROJECT_NAME=pipeline
 
 # postgrest and postgresql: required environment variables for 'gamma', prod and 'local'
-POSTGRES_USER=root
-POSTGRES_PASSWORD=supercoolpassword
+POSTGRES_USER=postgres
```
Comment (lorerod) [inline env.template:5]: Perhaps it would be best to maintain this as the `root` user.

File: scripts/config.ts [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
 import path from "path";
 
-export const COUCHDB_DBS = process.env.COUCHDB_DBS || "couchdb";
+export const COUCHDB_DBS = process.env.COUCHDB_DBS || "medic";
 
 export const POSTGRES = {
-  username: process.env.POSTGRES_USER || 'root',
-  password: process.env.POSTGRES_PASSWORD || 'supercoolpassword',
-  db: process.env.POSTGRES_DB || 'db',
+  username: process.env.POSTGRES_USER || 'postgres',
```
Comment (lorerod) [inline scripts/config.ts:7]: same here

File: tests/e2e-test.spec.ts
Comment (lorerod) [inline tests/e2e-test.spec.ts:1]: Thanks!

**Review by lorerod — CHANGES_REQUESTED** [review]
Hi Andra! I'm having problems when running the test locally and changing postgres user from `root` to anything else.

File: .github/workflows/test.yml (hunk 1 above)
Comment (andrablaj) [reply .github/workflows/test.yml:10]: > Role "root" does not exist.

@lorerod When you try `postgres` user does it work? As I removed the `root`, and added `postgres` instead.

Comment (lorerod) [reply .github/workflows/test.yml:10]: Yes with root it works

Comment (andrablaj) [reply .github/workflows/test.yml:10]: Can you paste your `.env` here, please?

Comment (lorerod) [reply .github/workflows/test.yml:10]:
```
# project wide: optional
COMPOSE_PROJECT_NAME=pipeline

# postgrest and postgresql: required environment variables for 'gamma', prod and 'local'
POSTGRES_USER=postgres
POSTGRES_PASSWORD=postgres
POSTGRES_DB=data
POSTGRES_TABLE=medic # for dbt use only
POSTGRES_SCHEMA=v1

# dbt: required environment variables for 'gamma', 'prod' and 'local'
DBT_POSTGRES_USER=postgres
DBT_POSTGRES_PASSWORD=postgres
DBT_POSTGRES_SCHEMA=dbt
DBT_POSTGRES_HOST=postgres # Your postgres instance IP or endpoint in "prod".
CHT_PIPELINE_BRANCH_URL="https://github.com/medic/cht-pipeline.git#hardcoded-variables"

# couchdb and logstash: required environment variables for 'gamma', 'prod' and 'local'
COUCHDB_USER=medic
COUCHDB_PASSWORD=password
COUCHDB_DBS="medic" # space separated list of databases you want to sync e.g "medic medic_sentinel"
COUCHDB_HOST=couchdb
COUCHDB_PORT=5984
COUCHDB_SECURE=false

# superset: required environment variables for 'gamma', 'prod' and 'local'
SUPERSET_PASSWORD=password
SUPERSET_ADMIN_EMAIL=user@superset.com
SUPERSET_SSH_TUNNELING=True
```

Comment (lorerod) [reply .github/workflows/test.yml:10]: You are right. I download all the images again, including the new `medicmobile/dataemon`; it is now working using postgres as user.
You can ignore my suggestion.

Comment (andrablaj) [reply .github/workflows/test.yml:10]: Perfect, thanks for confirming Lorena! I will update the branch to `main`, and send the PR as ready for review.

**Review by lorerod — APPROVED** [review]
Thanks!

**Pull request merged**